packages/docs/ssr/nuxt.md

```diff
@@ -25,6 +25,16 @@ export default defineNuxtConfig({
 
 And that's it, use your store as usual!
 
+:::tip 
+If you're using **npm**, `ERESOLVE unable to resolve dependency tree` error may happen when trying to `npm install` pinia into Nuxt project. This can be fixed by adding the following into your `package.json`:
+
+```js
+"overrides": { 
+     "vue": "latest"
```

Comment:
I think it should be:


```suggestion
  "vue": "latest"
```

Reply:
moved the section above + changed ident to 2 spaces to match proper npm json format